common/web/lm-worker/build-wrapper.js

```diff
@@ -0,0 +1,108 @@
+import fs from 'fs';
+
+import convertSourcemap from 'convert-source-map'; // Transforms sourcemaps among various common formats.
+                                                   // Base64, stringified-JSON, end-of-file comment...
+
+let INCLUDE_SRCMAPS = false;
+
+let sourceFromArgs;
+let destFromArgs;
+
+function doHelp(errCode) {
```

Comment:
Suggest making the same change as in build-polyfiller.js

Reply:
```suggestion
function doHelp(errMessage) {
  if(errMessage) {
    console.error(errMessage + '\n');
  }

```